### PR TITLE
create api integration done and added cart

### PR DESCRIPTION
**Pull request by Nikhil5922**
create api integration done and added cart

### CODE REVIEW

File: frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/hooks/ads/useADSCreateAPI.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { useQuery, useMutation } from "react-query";
+
+import { ADSServices } from "../../services/elements/ADS"
+
+
+
+export const useADSCreateAPI = (tenantId, type = true) => {
```
Comment (Keshav-NIUA) [inline frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/hooks/ads/useADSCreateAPI.js:7]: add comment here

File: frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/services/elements/ADS.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import Urls from "../atoms/urls";
+import { Request } from "../atoms/Utils/Request";
+
+ 
+export const ADSServices= {
```
Comment (Keshav-NIUA) [inline frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/services/elements/ADS.js:5]: add comment about component

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/pages/citizen/Create/ADSAcknowledgement.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      applicationNumber={props.data?.bookingApplication[0].bookingNo}
+      info={props.isSuccess ? props.t("ADS_BOOKING_NO") : ""}
+      successful={props.isSuccess}
+      style={{ width: "100%" }}
+    />
+  );
+};
+
+
+const ADSAcknowledgement = ({ data, onSuccess }) => {
```
Comment (Keshav-NIUA) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/pages/citizen/Create/ADSAcknowledgement.js:40]: add comment here

File: frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/hooks/ads/useADSCreateAPI.js (hunk 1 above)
Comment (Keshav-NIUA) [inline frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/hooks/ads/useADSCreateAPI.js:7]: add comment here

File: frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/services/elements/ADS.js (hunk 2 above)
Comment (Keshav-NIUA) [inline frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/services/elements/ADS.js:5]: add comment here about the component

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/components/ADSCartDetails.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useState } from "react";
+import ApplicationTable from "./ApplicationTable";
+import { CardSubHeader, DeleteIcon, Modal } from "@nudmcdgnpm/digit-ui-react-components";
+import { useLocation } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+
+const Close = () => (
```
Comment (Keshav-NIUA) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/components/ADSCartDetails.js:7]: add comment here

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/components/ADSCartDetails.js [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+  const handleDelete = (index) => {
+    const updatedList = data.filter((_, idx) => idx !== index);
+    setData(updatedList);
+  };
+
+  const calculateTotalPrice = () => {
+    return data.reduce((total, item) => total + Number(item.total_price), 0);
+  };
+
+  const columns = [
```
Comment (Keshav-NIUA) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/components/ADSCartDetails.js:53]: add comment here about every function as suggested

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/pageComponents/ADSearch.js [hunk 6]
```diff
@@ -329,13 +280,40 @@ console.log("dddd",selectNight);
     }
   };
 
+  const handleCartClick= () => {
```
Comment (Keshav-NIUA) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/pageComponents/ADSearch.js:283]: add description of function

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/utils/index.js [hunk 7]
```diff
@@ -26,3 +26,95 @@ export const pdfDownloadLink = (documents = {}, fileStoreId = "", format = "") =
...
+      
+    };
+  
+    data.documents = doc;
+    return data;
+  };
+
+
+
+export const ADSDataConvert = (data) => {
```
Comment (Keshav-NIUA) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/utils/index.js:89]: add comment here

**Review by Keshav-NIUA — COMMENTED** [review]
Make the changes and resolve comments

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/pages/citizen/Create/ADSAcknowledgement.js (hunk 3 above)
Comment (Nikhil5922) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/pages/citizen/Create/ADSAcknowledgement.js:40]: Done

File: frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/hooks/ads/useADSCreateAPI.js (hunk 1 above)
Comment (Nikhil5922) [reply frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/hooks/ads/useADSCreateAPI.js:7]: done

File: frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/services/elements/ADS.js (hunk 2 above)
Comment (Nikhil5922) [reply frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/services/elements/ADS.js:5]: done

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/components/ADSCartDetails.js (hunk 4 above)
Comment (Nikhil5922) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/components/ADSCartDetails.js:7]: done

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/components/ADSCartDetails.js (hunk 5 above)
Comment (Nikhil5922) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/components/ADSCartDetails.js:53]: done

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/pageComponents/ADSearch.js (hunk 6 above)
Comment (Nikhil5922) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/pageComponents/ADSearch.js:283]: done

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/utils/index.js (hunk 7 above)
Comment (Nikhil5922) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/utils/index.js:89]: done

**Pull request merged**